tema2/src/App.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import {Phone} from './components/Phone.jsx';
+import {Cover} from './components/Cover.jsx';
+import './App.css';
+
+export default class App extends React.Component{
+   constructor(props) {
+      super(props);
+      this.state = {
+        phones:[
+          {imageUrl: require('./phones/iphone11.png'), name:'iPHONE 11', price: 5400, stock:'In stock'},
+          {imageUrl: require('./phones/huaweimate30.png'), name:'Huawei Mate 30', price: 3900},
+          {imageUrl: require('./phones/samsungnote10.png'), name:'Samsung Note 10', price: 5899, stock: 'In stock'},
+          {imageUrl: require('./phones/sonyxperia1.png'), name: 'Sony Xperia 1', price:3399},
+          {imageUrl: require('./phones/blackberrykey2.png'), name: 'Blackberry Key2', price: 2980}
+        ],
+        covers:[
+          {imageUrl: require('./covers/coverhuawei.jpg'), name:'Huawei leather cover', price: 120},
+          {imageUrl: require('./covers/coveriphone.jpg'), name:'iPHONE 11 leather cover', price: 140},
+          {imageUrl: require('./covers/coverkey2.jpg'), name:'Blackberry leather cover', price: 99},
+          {imageUrl: require('./covers/covernote10.jpg'), name: 'Note 10 leather cover', price:150, stock:'In stock'},
+          {imageUrl: require('./covers/coverxz2.jpg'), name: 'Sony Xperia leather cover', price: 110, stock: 'In stock'}
+        ]
+      }
+   }
+
+     renderPhones = () => {
+       return  this.state.phones.map(phone => <Phone 
+        imageUrl={phone.imageUrl}
```

Comment:
Sa nu uiti de proprietatea `key={cheie_unica}`. Este foarte important ca atunci cand afisezi mai multe elemente de acelasi tip, cum ar fi o lista, sa folosesti proprietatea speciala `key` pentru a evita randari inutile.
Mai multe detalii gasesti aici: https://reactjs.org/docs/lists-and-keys.html